src/dune_lang/string_with_vars.mli

```diff
@@ -10,6 +10,8 @@ open Dune_util
 (** A sequence of text and variables. *)
 type t
 
+val pforms : t -> Pform.t list
```

Comment:
Where is this function being used?

Reply:
Yeah Previous commit, I did forgot. I'll remove it.
Thanks.